### DISCUSSION TITLE
Any example with webtechnolgies like mvc or webapi

### DISCUSSION THREAD

**Discussion by sachin0606** [dotnet/WatsonWebserver#54] [Q&A]
First I would like to thank you for the tool. 

I need to know how can i integrate it in current web application of mine 

Regards, 
Sachin

**Comment by jchristn** [comment]
Hi @sachin0606 i don't have examples with those frameworks but I would encourage you to look at the Test project for a working example.

**Comment by sachin0606** [comment]
Hi @jchristn sure i am working on the project so surely update on this.
